### PR TITLE
PXP-481: [Wukong CLI] Telemetry panic when the `~/.config/wukong` path is not exists 

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-481](https://mindvalley.atlassian.net/browse/PXP-481)

### Fixed
- [x] fix the panic for telemetry when the `~/.config/wukong` directory doesn't exist 
- [x] fix the not consistent error message from `wukong config` command
 Before:
Screenshot 2023-02-21 at 9 10 04 PM
After:
Screenshot 2023-02-21 at 9 30 41 PM

[PXP-481]: https://mindvalley.atlassian.net/browse/PXP-481?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/65#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Pull request merged**
